<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFilter2dPolar</class>
 <widget class="QMainWindow" name="frmFilter2dPolar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2D方向图设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxDomainPF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>切面</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbTheta">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>68</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Theta</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbPhi">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>68</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Phi</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>切面值</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxValues">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
